Jenkinsfile

```diff
@@ -35,10 +43,12 @@ pipeline {
         }
         stage("Build & Publish Artifacts") {
             when {
-                expression { !isPR }
+                expression { isPR }
```

Comment:
is this change intentional?

Reply:
Yes to test the Jenkinsfile ;) I reverted.